app/controllers/graphql_controller.rb

```diff
@@ -30,4 +30,31 @@ def ensure_hash(ambiguous_param)
       raise ArgumentError, "Unexpected parameter: #{ambiguous_param}"
     end
   end
+
+  def current_user
+    if doorkeeper_token && !doorkeeper_token.expired?
+      return User.find(doorkeeper_token.resource_owner_id)
+    end
+
+    raise Doorkeeper::Errors::DoorkeeperError, "Invalid access token"
+  end
+
+  # :reek:FeatureEnvy
+  def doorkeeper_unauthorized_render_options(error: nil)
+    return unless error
+
+    {
+      json: {
+        errors: [
+          {
+            message: error.description,
+            extensions: {
+              code: error.name,
+              state: error.state
+            }
+          }
+        ],
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/d439434a286bd76fead4ad367fe89e1a1cb8da1b/app/controllers/graphql_controller.rb#L56"></span>Avoid comma after the last item of a hash.</td>
    </tr>
  </tbody>
</table>